===================================================================
RCS file: /cvs/src/src/winsup/cygwin/include/fenv.h,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- src/winsup/cygwin/include/fenv.h	2010/09/11 06:53:28	1.1
+++ src/winsup/cygwin/include/fenv.h	2011/04/24 11:54:06	1.2
@@ -9,6 +9,7 @@
 details. */
 
 #ifndef _FENV_H_
+#define _FENV_H_
 
 #ifdef __cplusplus
 extern "C" {
